near-sdk/src/collections/legacy_tree_map.rs

```diff
@@ -18,6 +20,7 @@ use crate::IntoStorageKey;
 /// - `above`/`below`:          O(log(N))
 /// - `range` of K elements:    O(Klog(N))
 ///
+#[deprecated(since = "4.1.0", note = "Use tree_map::TreeMap instead")]
```

Comment:
Unfortunately, it's not that simple. Just swapping to use this type is a state-breaking change. At least the note should indicate this.

Also, the `tree_map` module is not exposed, so you might want to change this to say `near_sdk::collections::TreeMap` instead.

Optionally, this PR could introduce some migration of the old collection to the new one as a method, for example something like `LegacyTreeMap::to_tree_map(&self)` or something like this and then this comment can point to this to make the migration easy. This is less important, though, as I presume no one is really using this and if they absolutely want it, they can just ignore the deprecation warnings

Reply:
So many mistakes in one line!